workflow/executor/pns/pns.go

```diff
@@ -263,6 +257,13 @@ func (p *PNSExecutor) getContainerPID(containerName string) int {
 	if pid, ok := p.containerNameToPID[containerName]; ok {
 		return pid
 	}
+	procs, _ := gops.Processes()
```

Comment:
`containerNameToPID`  might not contain the container, lets look at the current processes